qiskit/visualization/matplotlib.py

```diff
@@ -26,6 +26,8 @@
 import numpy as np
 
 try:
+    import matplotlib
+    matplotlib.use('ps')
```

Comment:
This is not something we can do here. `use()` will override any actual environment settings: https://matplotlib.org/faq/usage_faq.html#what-is-a-backend

We can not force users to use a specific backend when running the visualization function. This will break a whole bunch of things that go to use mpl visualizer. This is literally just needed for image comparison testing and not something we can force in general.